sql-cli/tests/test___main__.py

```diff
@@ -20,7 +23,7 @@ def get_stdout(result) -> str:
 def test_about():
     result = runner.invoke(app, ["about"])
     assert result.exit_code == 0
-    assert "Find out more: https://github.com/astronomer/astro-sdk/sql-cli" == get_stdout(result)
+    assert "Find out more" in get_stdout(result)
```

Comment:
Why did you change this?

Reply:
This was a poor rebasing merge - I'll rollback it.